pkg/ocm/ocm.go

```diff
@@ -17,8 +16,14 @@ import (
 	hivev1 "github.com/openshift/hive/apis/hive/v1"
 )
 
-var (
-	sl_clusterHasGoneMissing = "https://raw.githubusercontent.com/openshift/managed-notifications/master/osd/cluster_has_gone_missing.json"
+const (
```

Comment:
can we put this in a struct actually ?

Reply:
We can do a const out of this, but a struct cannot be const as far as I know :thinking: 
Or is `struct` a typo? :D

Reply:
Hmm that is true we would need a helper function to create the struct. Maybe you can add NewClusterMissingSL function

Reply:
I don't understand the benefit of having a struct there, yet. Can you explain this?
If we make a struct out of this the struct will be mutable again and then we will have the G107 security alert again, because the URL field will be not a const anymore.


Maybe we should decide first if we want to embed or not embed. This PR is embedding the URL, but embedding it comes with a price mentioned above.